### PR TITLE
Fixed Structure & made login/signup nav consistent to other pages

### PR DESCRIPTION
**Pull request by ShrutiLakhani**

### CODE REVIEW

**Comment by ShrutiLakhani** [comment]
Please review.  I have fixed file structure and also fixed minor issues in header for login/signup page

File: Cart/Cart.html [hunk 1]
```diff
@@ -6,142 +6,144 @@
...
-                                    <!-- <div class="img-tint"></div> -->
-
+                                    <div class="card-details">
+                                        <h3>Michael Kors</h3>
+                                        <p>Hamilton Legacy Large Logo Belted Satchel</p>
+                                        <p>Brown<p>
+                                                <div class="quantity">
+                                                    <label for="quantity">Qty
+                                                        <input class="cart-quantity" type="number" id="quantity"
+                                                            name="quantity" min="1" max="5" placeholder="1">
```
Comment (Debarshi95) [inline Cart/Cart.html:101]: A suggestion instead of nested divs, you can use semantic tags liks section, article etc.

**Review by Debarshi95 — APPROVED** [review]
Looks good. Try to use section and article tags instead of creating div soup. Rest looks good.

File: Cart/Cart.html [hunk 2]
```diff
@@ -6,142 +6,144 @@
...
+                                            alt="bag-image" />
+                                    </div>
+                                    <div class="card-details">
+                                        <h3>Michael Kors</h3>
+                                        <p>MICHAEL Brown & Acorn Carmen Small Satchel</p>
+                                        <p>Brown<p>
+                                                <div class="quantity">
+                                                    <label for="quantity">Qty
+                                                        <input class="cart-quantity" type="number" id="quantity"
+                                                            name="quantity" min="1" max="5" placeholder="1">
```
Comment (swethaogeti) [inline Cart/Cart.html:76]: reset the code

File: WishList/Wishlist.html [hunk 3]
```diff
@@ -6,123 +6,124 @@
...
-                            <a href="/Cart/Cart.html" class="btn-bag-card">MOVE TO BAG</a>
-                        </div>
+                    <!-- Card with Dismiss -->
+                    <div class="card2-container wishlist-card">
+                        <span class="closebtn">&times;</span>
+                        <img src="https://img.tatacliq.com/images/i7/658Wx734H/MP000000009138731_658Wx734H_202103222027121.jpeg"
+                            alt="image-bag" />
+                        <h1>Michael Kors</h1>
+                        <h2>MICHAEL Michael Kors Brown</h2>
+                        <h5><ins>₹29,500</h5>
```
Comment (swethaogeti) [inline WishList/Wishlist.html:81]: this is good . i will try using ins tag

**Review by anshulraheja — APPROVED** [review]
UI is clean. Just one thing, add some margin between discount amount and original amount inside the card on the wishlist page

File: index.html [hunk 4]
```diff
@@ -15,72 +15,77 @@
...
+                        <a href="/Product/Product.html"><img
+                                src="https://assets.tatacliq.com/medias/sys_master/images/32285859217438.jpg"
+                                alt="image-watch" /></a>
+                    </div>
+                    <div class="card2-container wishlist-card">
+                        <a href="/Product/Product.html"><img
+                                src="https://assets.tatacliq.com/medias/sys_master/images/32285859414046.jpg"
+                                alt="image-footwear" /></a>
+                    </div>
+                    <div class="spacer-3rem"></div>
```
Comment (swethaogeti) [inline index.html:87]: everything looks good just rest the code. GTM!

**Review by swethaogeti — APPROVED** [review]
UI looking amazing. good to merge

**Pull request merged**